### ISSUE TITLE
vscode cannot help installing packages (e.g.: ipykernel for notebook)

### ISSUE THREAD

**Issue by Julian-J-S** [astral-sh/uv#1601]
vs code cannot help installing packages as it used to

example:
- `uv venv`
- create a python notebook `dummy.ipynb` (use the newly created .venv)
- run any cell

vscode will prompt 
<img width="257" alt="image" src="https://github.com/astral-sh/uv/assets/25177421/9ef24d9c-88db-4645-9fcd-fc098e96f070">

if you click on "install" this usually works but as no `pip` is available you get
<img width="257" alt="image" src="https://github.com/astral-sh/uv/assets/25177421/a49731a8-ca91-4ab9-ad88-355e292c2423">

Ofc there are solutions like adding `pip` or installing it manually via `uv pip install ipykernel` but the usual workflow is broken. Not sure if another vscode extension would solve the problem or any other ideas.

anyway, still LOOOOVEEEE all your great, fast, ergonomic python tooling and looking forward for even more! ❤️

**Comment by kafonek** [comment]
As a workaround (or solution?), you can `uv venv --seed` to install pip, setuptools, and wheel into the virtual env that `uv` creates.

**Comment by casellimarco** [comment]
I don't know if it is related, but when running jupyter notebooks I struggle to update the venv where the kernel has been loaded. 
For instance, if a block fails because of the absence of a package, I could not find a way to fix it, seems "Restart kernel" is not pointing towards the latest version of the virtual environment. The funny thing is that `pip list` finds the package.
![image](https://github.com/astral-sh/uv/assets/22890646/284f7908-758d-4b93-9ef9-f9b56d31c225)

**Comment by kafonek** [comment]
@casellimarco for your own debug you can try `!which pip` or `!pip install -v pandas`. Both should give you a clue about which pip you're using. I believe you're right that `!uv pip install` works right now (as long as you started `jupyter` from the `uv` venv)

It would be nice to have something akin to `poetry run` here which sources the venv before starting the process.

```
poetry init -n
poetry add jupyter notebook
poetry run jupyter notebook

# in notebook, shows the venv pip
!which pip
```

```
uv venv --seed
uv pip install jupyter notebook
./.venv/bin/jupyter notebook # or ./.venv/bin/python -m jupyter notebook

# in notebook, shows not-venv pip
!which pip
```

explicitly sourcing the venv is a workaround for you @casellimarco 

```
uv venv --seed
source .venv/bin/activate
jupyter notebook

# in notebook, shows venv pip
!which pip
```

**Comment by casellimarco** [comment]
Thanks for the help @kafonek!

**Comment by mattharrison** [comment]
Not sure how to get this working. I added pip and ipykernel to my uv venv and am still getting this popup.

**Comment by RoySelaArazi-Q** [comment]
This is still isn't solved to me :(

Tried to create a new venv by running:
```
uv venv test --seed 
source test/bin/activate
uv pip install ipykernel
```

Still vscode gives the same error when trying to use this environment.

**Comment by RoySelaArazi-Q** [comment]
This is still isn't solved to me :(

Tried to create a new venv by running:
```
uv venv test --seed 
source test/bin/activate
uv pip install ipykernel
```

Still vscode gives the same error when trying to use this environment.
